Microsoft.Toolkit.Uwp.UI.Controls/ControlHelpers.DesignTime.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System.Reflection;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
```

Comment:
Name space should be 
Microsoft.Toolkit.Uwp not Microsoft.Toolkit.Uwp.Controls like the other ControlHelpers.Composition.cs & ControlHelpers.Math.cs

Reply:
Huh? This is for UI Controls only, and part of a partial class that is already in that namespace.

Reply:
![image](https://user-images.githubusercontent.com/6191377/30301602-13b75204-979f-11e7-97bd-0d01ce761853.png)
![image](https://user-images.githubusercontent.com/6191377/30301608-17dff05c-979f-11e7-8d59-a66dbee58b91.png)

The other partial class are not on same namespace :) I guess we either move all of them or make this one similar to others

Reply:
Sorry I didn't intend to show my desktop background :P

Reply:
Ugh! Good catch! TBH I think the other classes are wrong. Since there are internal, we could update those too.

Reply:
Yep, other classes should be updated as they should be in Controls namespace :)